app/i18n/community_translators.md

```diff
@@ -0,0 +1,125 @@
+
+The advanced translation flow requires setting up a local building infrastructure for Decrediton. This setup is more involved, recommended when the translator is a software developer (and thus already have experience doing these kinds of setups) or is a self-sufficient technical person and can figure most things by themselves. The main advantage is that this method allows doing translations at any time during development and to immediately test any necessary adjustments to the translation.
+
+The next sections detail both translation flows.
+
+## Basic UI Translation
+
+Basic UI translation can be done once an RC (release candidate) version of Decrediton is released.
+
+Translators have to fulfill the following requirements:
+
+- Create a [Github](https://github.com) account
+- Install [git software](https://guides.github.com/introduction/git-handbook/)
+- Download the latest released Decrediton RC version
+- Follow the [#decrediton](https://matrix.to/#/#decrediton:decred.org) matrix chat room for announcements
+
+Do note this document does not attempt to explain how git or Github works, so at least some basic knowledge about those is expected. See the links above for some starting points.
+
+To generate a new translation file for inclusion in the next release, follow the following procedure.
+
+### Check out the most recent Decrediton source:
+
+To setup the repository for the first time, fork it in the Github interface, then clone it:
+
+```shell
+$ git clone git@github.com:<github-username>/decrediton
+$ cd decrediton
+$ git remote add upstream https://github.com/decred/decrediton
+$ git fetch upstream
+```
+
+When updating an already existing install, fetch the latest changes. **Important**: this will **discard** any local changes. Those changes will be lost **forever**.
+
+```shell
+$ git fetch upstream
+$ git reset --hard
+```
+
+### Create a new working branch
+
+This updates the working copy and creates a new branch for the translation work. Replace `[new-decrediton-version]` with the new version under development.
+
+```shell
+$ git checkout -b translations-v[new-decrediton-version] upstream/master
+```
+
+
+### Use the Translation UI
+
+Open the file `decrediton-install-dir/app/i18n/translator.html` in a browser. This can be done by opening a modern browser, typing `file:///` in the address bar, then navigating to the dir where Decrediton source code as checked out.
+
+That file contains a very simple UI for working with Decrediton translation files.
+
+Load the following files, all located in the checked out copy of the repository, in the `app/i18n/translations` subdir:
+
+- `previous_original.json` (this file contains the original strings of the _previous_ Decrediton version)
```

Comment:
It is not clear from this doc how this file is involved. But I haven't checked `translator.html`, perhaps it is explained there. If so, ignore this comment.

Reply:
It's used to determine source strings that changed between releases (hypothetical example, `app.title` changed from `Decrediton` to `Here's Decrediton`). 

There aren't really "explanations" in the translator file (it's pretty rough for the moment) but it shouldn't really matter as long as the translator loads the file.